Dockerfile.profiles

```diff
@@ -0,0 +1,10 @@
+# Go build
```

Comment:
We weren't sure where folks would want the Dockerfile to live. How would folks feel about us creating a `build/` directory, containing the two docker files? Or any other suggestions?